### ISSUE TITLE
Download Request Error

### ISSUE THREAD

**Issue by smaragdus** [erickutcher/httpdownloader#70]
When I tried to download an image from [Minds](https://www.minds.com/) I got an error message:

`An error occurred while sending the download request.`

**screens:**

![HTTP Downloader 1 0 2 6 - 2019-08-01 - error - 001](https://user-images.githubusercontent.com/9025757/62252667-2ecf9f80-b3fc-11e9-99bc-034d9b3cff8d.png)

![HTTP Downloader 1 0 2 6 - 2019-08-01 - error - 002](https://user-images.githubusercontent.com/9025757/62252675-35f6ad80-b3fc-11e9-8bb9-f5fdb9400447.png)

- [link](https://cdn.minds.com/fs/v1/thumbnail/997352051191111680/xlarge)

I was able to download the image via the browser (Chromium-based) temporarily disabling browser integration.

Please confirm the problem. Or perhaps this happens only with my setup?

**Comment by erickutcher** [comment]
Works fine in Firefox and Vivaldi. That error means that the extension can't communicate with the program's web server. Make sure the server is active. A restart of the program should fix it if it just happened to disconnect.

**Comment by smaragdus** [comment]
@erickutcher 

Thanks for the explanation. Before I posted this issue I tried to get the image 3 times via HTTP Downloader and it failed (I was supposing it might be cookies-related). After I read your post I restarted the program and the image file was downloaded but without extension, **xlarge** in stead of **xlarge.jpg**. Did you get the file with extension? Perhaps my setup is messed up. I am closing this issue. Thanks for the advice.
